servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.h

```diff
@@ -236,8 +252,78 @@ class RenderSceneBuffersRD : public RenderSceneBuffers {
 	RID get_velocity_buffer(bool p_get_msaa, uint32_t p_layer);
 
 	////////////////////////////////////////////////////////////////////////////////////////////////////////////
-	// Everything after this needs to be re-evaluated, this is all old implementation
+	// Our classDB doesn't support calling our normal exposed functions
+
+private:
+	RID _create_texture_from_format(const StringName &p_context, const StringName &p_texture_name, const Ref<RDTextureFormat> &p_texture_format, const Ref<RDTextureView> &p_view = Ref<RDTextureView>(), bool p_unique = true);
+	RID _create_texture_view(const StringName &p_context, const StringName &p_texture_name, const StringName p_view_name, const Ref<RDTextureView> p_view = Ref<RDTextureView>());
+	Ref<RDTextureFormat> _get_texture_format(const StringName &p_context, const StringName &p_texture_name) const;
+
+	// For color and depth as exposed to extensions, we return the buffer that we're rendering into.
+	// Resolving happens after effects etc. are run.
```

Comment:
Is there a way for the user to grab the resolved buffer?

Reply:
Yeah they can access them directly through `get_texture`, but the important bit here is that they won't be resolved until after the effect code is run (which right now is the last part where we're currently giving access to this).

It might also be nice to add a `list_textures` method so people can see whats available BUT since all buffers are destroyed when the viewport resizes and only some are re-created right away (and others are created when first needed) that might be a method that gives the wrong impression.